src/cli/rustup_mode.rs

```diff
@@ -1025,16 +1025,17 @@ fn doc(cfg: &Cfg, m: &ArgMatches<'_>) -> Result<()> {
 }
 
 fn man(cfg: &Cfg, m: &ArgMatches<'_>) -> Result<()> {
-    let manpage = m.value_of("command").expect("");
+    let command = m.value_of("command").unwrap();
     let toolchain = explicit_or_dir_toolchain(cfg, m)?;
-    let mut man_path = toolchain.path().to_path_buf();
-    man_path.push("share");
-    man_path.push("man");
-    man_path.push("man1");
-    man_path.push(manpage.to_owned() + ".1");
-    utils::assert_is_file(&man_path)?;
+    let mut manpaths = toolchain.path().to_path_buf();
+    manpaths.push("share");
+    manpaths.push("man");
+    utils::assert_is_directory(&manpaths)?;
```

Comment:
Is it necessary to check for existence of the man dir?
Can we assume that Rustup always install manpages in this location?

Reply:
I think it's not unreasonable to fail more informatively if someone types `rustup man` and the manpages moved, the error might be more helpful than a simply missing or incorrect manpage